L1Trigger/TrackFindingTracklet/interface/TrackBuilderChannel.h

```diff
@@ -0,0 +1,35 @@
+#ifndef L1Trigger_TrackFindingTracklet_TrackBuilderChannel_h
+#define L1Trigger_TrackFindingTracklet_TrackBuilderChannel_h
+
```

Comment:
Add comment to start of class explaining what it is. e.g. "Class to assign tracklet tracks to channel". (And comment at start of ProducerTrackBuilderChannel" should be "Creates TrackBuilderChannel class that assigns tracklet tracks to channel").

Reply:
done